docs/core/resilience/http-resilience.md

```diff
@@ -0,0 +1,215 @@
+
+### The standard hedging handler
+
+The standard hedging handler wraps the execution of the request with a standard hedging mechanism. The standard hedging handler is added by calling the `AddStandardHedgingHandler` extension method.
+
+```csharp
+// A service collection is manually instantiated for this example.
+// If you're using a generic host, you'll instead use its service collection:
+//
+// var host = Host.CreateApplicationBuilder(args);
+// host.Services.AddHttpClient<ExampleClient>(client => { ... });
+
+var services = new ServiceCollection();
+
+IHttpClientBuilder builder = services.AddHttpClient<ExampleClient>(
+    configureClient: static client =>
+    {
+        client.BaseAddress = new("https://jsonplaceholder.typicode.com");
+    });
+```
+
+The preceding code:
+
+- Adds an <xref:System.Net.Http.HttpClient> for the `ExampleClient` type to the service container.
+- Configures the <xref:System.Net.Http.HttpClient> to use `"https://jsonplaceholder.typicode.com"` as the base address.
+- Adds the standard hedging resilience handler to the <xref:System.Net.Http.HttpClient>.
+
+The `ExampleClient` is defined as follows:
+
+:::code source="snippets/http-resilience/ExampleClient.cs":::
+
+The preceding code:
+
+- Defines an `ExampleClient` type that has a constructor that accepts an <xref:System.Net.Http.HttpClient>.
+- Exposes a `GetCommentsAsync` method that sends a GET request to the `"/comments"` endpoint and returns the response.
+
+The `Comment` type is defined as follows:
+
+:::code source="snippets/http-resilience/Comment.cs":::
+
+#### Standard hedging handler defaults
+
+The standard hedging uses a pool of circuit breakers to ensure that unhealthy endpoints are not hedged against. By default, the selection from the pool is based on the URL authority (scheme + host + port).
+
+> [!TIP]
+> It's recommended that you configure the way the strategies are selected by calling `StandardHedgingHandlerBuilderExtensions.SelectPipelineByAuthority`.
+
+To use the standard hedging handler, call `AddStandardHedgingHandler` extension method. The following example configures the `ExampleClient` to use the standard hedging handler.
+
+```csharp
+builder.AddStandardHedgingHandler();
+```
+
+The preceding code, adds the standard hedging handler to the <xref:Microsoft.Extensions.DependencyInjection.IHttpClientBuilder>. The default configuration chains five resilience strategies in the following order (from the outermost to the innermost):
+
+| Order | Strategy | Description |
+|--:|--|--|
+| **1** | Total request timeout | Total request timeout pipeline applies an overall timeout to the execution, ensuring that the request including hedging attempts, does not exceed the configured limit. |
+| **2** | Hedging | The hedging strategy executes the requests against multiple endpoints in case the dependency is slow or returns a transient error. |
```

Comment:
```suggestion
| **2** | Hedging | The hedging strategy executes the requests against multiple endpoints in case the dependency is slow or returns a transient error. |
```

The routing is optional, by default it just hedges agains the url provided in the original HttpRequestMessage